ui/selectors/selectors.js

```diff
@@ -348,7 +348,7 @@ export const getMetaMaskAccountsConnected = createSelector(
 
 export function isBalanceCached(state) {
   const selectedAccountBalance =
-    state.metamask.accounts[getSelectedAddress(state)].balance;
+    state.metamask.accounts[getSelectedAddress(state)]?.balance;
```

Comment:
Hmm, this change doesn't have any obvious connection to the rest of the PR

Reply:
🤔 weird. Removing!

Reply:
removed here: https://github.com/MetaMask/metamask-extension/pull/17421/commits/ccfc91787ab01012269c2d2bd97fa9b3c0de5ace#diff-45ae84764c1d49a07813a91adc3351d6ed96bf9ee8fa2ae4c9a116d1e0cba06cR351